profiler/profiler.go

```diff
@@ -154,6 +158,15 @@ func (p *profiler) run() {
 	if _, ok := p.cfg.types[BlockProfile]; ok {
 		runtime.SetBlockProfileRate(p.cfg.blockRate)
 	}
+	if s, ok := p.cfg.statsd.(statsdGauge); ok {
```

Comment:
Hacky ... but adding `Gauge` to `StatsdClient` is a potentially breaking API change ... so yeah – this is the right call!

<img src="https://user-images.githubusercontent.com/15000/151519162-5966e0ce-5a80-4b33-85f5-26c050dc0283.png" height="400"/>